backend/src/main/java/com/huellapositiva/domain/service/ESALContactPersonService.java

```diff
@@ -53,4 +57,10 @@ public Id registerContactPerson(PlainPassword plainPassword, EmailConfirmation e
     public Integer updateJoinedESAL(JpaContactPerson jpaContactPerson, JpaESAL jpaESAL) {
         return esalContactPersonRepository.updateESAL(jpaContactPerson.getId(), jpaESAL);
     }
+
+    public void validations(String phone) {
```

Comment:
Method names should be verbs to make the program easier to understand and follow conventions.

Reply:
It's done.